modular_skyrat/modules/indicators/code/combat_indicator.dm

```diff
@@ -31,7 +31,22 @@ GLOBAL_VAR_INIT(combat_indicator_overlay, GenerateCombatOverlay())
 			nextcombatpopup = world.time + COMBAT_NOTICE_COOLDOWN
 			playsound(src, 'sound/machines/chime.ogg', 10, ignore_walls = FALSE)
 			flick_emote_popup_on_mob("combat", 20)
-			visible_message("<span class='warning'><b>[src] gets ready for combat!</b></span>")
+			var/ciweapon
+			if(get_active_held_item())
+				ciweapon = get_active_held_item()
+				if(istype(ciweapon, /obj/item/gun))
+					visible_message(span_warning("<b>[src] raises \the [ciweapon] with their finger on the trigger, ready for combat!<b>"))
+				else
+					visible_message(span_warning("<b>[src] readies \the [ciweapon] with a tightened grip and offensive stance, ready for combat!<b>"))
+			else
+				if(issilicon(src))
+					visible_message(span_warning("<b>[src] shifts its armour plating into a defensive stance, ready for combat!</b>"))
+				if(ishuman(src))
+					visible_message(span_warning("<b>[src] raises [p_their()] fists in an offensive stance, ready for combat!</b>"))
+				if(isalien(src))
+					visible_message(span_warning("<b>[src] hisses in a terrifying stance, claws raised and ready for combat!</b>"))
+				else
+					visible_message(span_warning("<b>[src] gets ready for combat!</b>"))
```

Comment:
We have the `span_boldwarning()` macro for this, we need to avoid manually using spans and other formatting as much as we can, to avoid issues like the ones we would've gotten because you forgot to close the `<b>` in like two cases (you forgot the `/`).
```suggestion
				if(istype(ciweapon, /obj/item/gun))
					visible_message(span_boldwarning("[src] raises \the [ciweapon] with their finger on the trigger, ready for combat!"))
				else
					visible_message(span_boldwarning("[src] readies \the [ciweapon] with a tightened grip and offensive stance, ready for combat!"))
			else
				if(issilicon(src))
					visible_message(span_boldwarning("<b>[src] shifts its armour plating into a defensive stance, ready for combat!"))
				if(ishuman(src))
					visible_message(span_boldwarning("[src] raises [p_their()] fists in an offensive stance, ready for combat!"))
				if(isalien(src))
					visible_message(span_boldwarning("[src] hisses in a terrifying stance, claws raised and ready for combat!"))
				else
					visible_message(span_boldwarning("[src] gets ready for combat!"))
```

Reply:
Good catch, didn't realise we had that (though I distinctly remember correctly closing my bolds, maybe I forgot to push 😔 )